rockcraft/commands/init.py

```diff
@@ -95,29 +95,54 @@ class InitCommand(AppCommand):
 
                 # to ensure the flask-framework extension works properly, your Flask application
                 # should have an `app.py` file with an `app` object as the WSGI entrypoint.
+                # A `requirements.txt` file with at least the flask package should also exist.
```

Comment:
```suggestion
                # a `requirements.txt` file with at least the flask package should also exist.
```

Reply:
If I have one incorrect, I have most of them :(

Reply:
I should've proposed fixes for all of them, but pls double check

Reply:
sure I will review the text after accepting your corrections